config/metadata/basic_metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# This file is blank to override base Hyku with the exception
+# of based_near. Hyrax does some magic that expects basic_metadata
+# to be included and contain based_near.
+
+# Each model will have its own model resource metadata.yaml
+# that includes basic metadata for more customizations
+
+attributes:
+  based_near:
+    type: string
+    multiple: true
+    form:
+      primary: false
+    index_keys:
+      - "based_near_sim"
+      - "based_near_tesim"
+    predicate: http://xmlns.com/foaf/0.1/based_near
```

Comment:
This might break things if we don't override the other work type's schemas... (GenericWork, Image, etc)